src/styles/main.scss

```diff
@@ -455,7 +455,7 @@ ul li.selected {
   }
 }
 
-#landingPage-searchInput {
+#landingPage-searchInput-cpr {
```

Comment:
Do we also need a `landingPage-searchInput-cclw` ?

Reply:
See:
```
src/components/forms/LandingSearchForm.tsx
70:          id={`landingPage-searchInput-${theme}`}
```

Reply:
The cpr specific input is because it is handling the animated placeholder effect we have on the CPR homepage